packages/app/cypress/e2e/specs.cy.ts

```diff
@@ -383,6 +383,67 @@ describe('App: Index', () => {
           .and('contain', defaultMessages.createSpec.e2e.importEmptySpec.header)
         })
       })
+
+      context('scaffold empty spec', () => {
+        it('should generate empty spec', () => {
+          cy.findByRole('button', { name: 'New Spec', exact: false }).click()
+
+          cy.findByRole('dialog', { name: defaultMessages.createSpec.newSpecModalTitle }).within(() => {
+            cy.findAllByTestId('card').eq(0)
+            .and('contain', defaultMessages.createSpec.e2e.importFromScaffold.header)
+
+            cy.findAllByTestId('card').eq(1)
+            .and('contain', defaultMessages.createSpec.e2e.importEmptySpec.header)
+          })
+
+          cy.contains('Create new empty spec').click()
+
+          cy.findAllByLabelText(defaultMessages.createSpec.e2e.importEmptySpec.inputPlaceholder)
+          .as('enterSpecInput')
+
+          cy.get('@enterSpecInput').invoke('val').should('eq', getPathForPlatform('src/filename.cy.js'))
+          cy.contains(defaultMessages.createSpec.e2e.importEmptySpec.invalidSpecWarning).should('not.exist')
+          cy.get('@enterSpecInput').clear()
+          cy.contains(defaultMessages.createSpec.e2e.importEmptySpec.invalidSpecWarning).should('not.exist')
+
+          // Shows entered file does not match spec pattern
+          cy.get('@enterSpecInput').type(getPathForPlatform('cypress/e2e/no-match'))
+          cy.contains(defaultMessages.createSpec.e2e.importEmptySpec.invalidSpecWarning)
+          cy.contains('button', defaultMessages.createSpec.createSpec).should('be.disabled')
+
+          cy.percySnapshot('Invalid spec error')
+
+          //Shows extension warning
+          cy.get('@enterSpecInput').clear().type(getPathForPlatform('cypress/e2e/MyTest.spec.j'))
+          cy.intercept('mutation-EmptyGenerator_MatchSpecFile', (req) => {
+            if (req.body.variables.specFile === getPathForPlatform('cypress/e2e/MyTest.spec.jx')) {
+              req.on('before:response', (res) => {
+                res.body.data.matchesSpecPattern = true
+              })
+            }
+          })
+
+          cy.get('@enterSpecInput').type('x')
+          cy.contains(defaultMessages.createSpec.e2e.importEmptySpec.specExtensionWarning)
+          cy.percySnapshot('Non-recommended spec pattern warning')
+          cy.contains('span', '{filename}.cy.jx')
```

Comment:
Is this the correct message? Wondering if the test should have been `cypress/e2e/MyTest.cy.js`
<img width="690" alt="Screen Shot 2022-03-22 at 9 51 25 AM" src="https://user-images.githubusercontent.com/14099737/159510755-e9152ec3-576b-4152-a255-a9b5ae357ddc.png">

Also looking while running looking and looking at this test, the spec file name generate (`cypress/e2e/*.spec.{js,jsx]}`) does not match the custom spec pattern (`src/**/*.cy.{js,jsx}`), however the test is passing.
<img width="724" alt="Screen Shot 2022-03-22 at 9 52 27 AM" src="https://user-images.githubusercontent.com/14099737/159510807-61424b83-1309-45f3-9187-77f17226d4d4.png">